ZConfig/components/logger/loghandler.py

```diff
@@ -25,6 +25,13 @@
 from logging.handlers import HTTPHandler, SMTPHandler
 from logging.handlers import NTEventLogHandler as Win32EventLogHandler
 
+# Export these, they're used in handlers.py
+SysLogHandler = SysLogHandler
+HTTPHandler = HTTPHandler
+SMTPHandler = SMTPHandler
+Win32EventLogHandler = Win32EventLogHandler
```

Comment:
Let's just hope linters won't decide to start warning about "useless assignment X = X" in the future ;)